dist/browser.js

```diff
@@ -1,194 +1,213 @@
-/*
```

Comment:
If easy, it'd be nice to keep the same order as before to minimize the diff and keep `var Rune = { version: ... }` at the top.

Reply:
Discussed in the voice chat. The order does not matter too much, since we expect users to read the api/source code and not the dist output.